source/genericobject/index.rst

```diff
@@ -49,8 +49,18 @@ Install the Plugin
 
 * Uncompress the archive.
 * Move the ``genericobject`` directory to the ``<GLPI_ROOT>/plugins`` directory
-* Navigate to the *Configuration > Plugins* page,
-* Install and activate the plugin.
+* Navigate to the *Configuration > Plugins* page
+* Install and activate the plugin
+* Set rights on plugin directories :
+    
+.. code-block:: bash
+   
+   chown -R myaccount:apache <GLPI_ROOT>/plugins/genericobject
+   chmod -R 750 <GLPI_ROOT>/plugins/genericobject
+
+   # Only when plugin activated
+   chown -R myaccount:apache <GLPI_ROOT>/files/_plugins/genericobject
```

Comment:
This hsould not be mandatory. ACLs on `files/_plugins/` are checked core side; and the subdirectory is certainly created from the plugin on its installation/activation (therefore ACLs are correct).

Reply:
You're right too. Rights on that directory after install : 
```
[~/glpi_plugins]: ls -rtl /var/www/glpi-bf/files/_plugins/
total 8
-rwxrwx--- 1 root     www-data   80 mars   6 02:36 remove.txt
drwxr-xr-x 8 www-data www-data 4096 mars   6 11:12 genericobject
```
`chmod`  is use to be more restrict on that directory (no rights for "others"). This prevents unauthorized access on files created by plugins with shell access user.

What do you want to do ?